qiskit/transpiler/passes/optimization/optimize_1q_commutation.py

```diff
@@ -58,7 +58,7 @@ class Optimize1qGatesSimpleCommutation(TransformationPass):
             barriers.)
     """
 
-    # NOTE: A run from `dag.collect_1q_runs` is always nonempty, so we sometimes use an empty list
+    # NOTE: A run from ``dag.collect_1q_runs`` is always nonempty, so we sometimes use an empty list
```

Comment:
Ok, the comments really don't need Sphinx formatting haha

Reply:
I do this sort of fixing while taking care of kiddo.. I tend to do it automatically, without a lot of thinking (and my IDE shows comments and docstrings in the same color) :) 

Addressed in 8b720781585c703d5a27fe0bebd708ae16fcf829